client/routes.jsx

```diff
@@ -21,7 +22,11 @@ const checkAuth = (store) => {
   store.dispatch(getUser());
 };
 
+// This short-circuit seems unnecessary - using the mobile <Switch /> navigator (future) should prevent this from being called
```

Comment:
Can we mark this comment as "TODO" so we can find it later?

Reply:
Will do!